packages/world/ts/library/config/types.ts

```diff
@@ -28,6 +29,24 @@ export type SystemUserConfig =
         }
     );
 
+export interface ExpandSystemConfig<T extends SystemUserConfig, SystemName extends string>
+  extends OrDefaults<
+    T,
+    {
+      name: SystemName;
+      registerFunctionSelectors: true;
+    }
+  > {
+  openAccess: T["openAccess"];
```

Comment:
default for `openAccess` should be true I think

Reply:
yep, but it wasn't reflected in zod or types, fixed